src/components/EnrolmentOfficerFormPanel.js

```diff
@@ -22,16 +22,15 @@ const styles = (theme) => ({
 const EnrolmentOfficerFormPanel = (props) => {
   const { edited, classes, modulesManager, onEditedChanged, readOnly } = props;
   const { formatMessage } = useTranslations("admin.EnrolmentOfficerFormPanel", modulesManager);
-
-  const isEnabled = edited.userTypes?.includes(ENROLMENT_OFFICER_USER_TYPE);
-  const hasRole = !!edited.roles ? edited.roles.filter((x) => x.isSystem == 1).length != 0 : false;
-  if (isEnabled) {
-    const {
-      isLoading,
-      data,
-      error: graphqlError,
-    } = useGraphqlQuery(
-      `
+  const [isEnabled, setIsEnabled] = useState(false);
+  const hasOfficerUserType = edited.userTypes?.includes(ENROLMENT_OFFICER_USER_TYPE);
+  const hasOfficerRole = edited.roles ? edited.roles.filter((x) => x.isSystem === 1).length !== 0 : false;
```

Comment:
I wonder if we can also shortcut this line of code by using 'optional chaining'
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
Change of this condition have other consequences, so I decided to keep it like that.